frontend/src/redux/reducers/viewers.js

```diff
@@ -0,0 +1,75 @@
+import CameraControls from '@metacell/geppetto-meta-ui/camera-controls/CameraControls';
+
+import {
+  ADD_INSTANCES_VIEWER, ADD_VIEWER, COLOR_INSTANCES_VIEWER, REMOVE_VIEWER,
+} from '../actions/viewers';
+
+const defaultCameraOptions = {
```

Comment:
Each global state > viewer besides instances will also manage cameraOptions

Reply:
seems reasonable, if later not needed in redux store we can also move it inside the Component state